e2e/src/api/specs/map.e2e-spec.ts

```diff
@@ -159,4 +159,75 @@ describe('/map', () => {
       expect(body).toEqual(expect.objectContaining({ id: 'immich-map-dark' }));
     });
   });
+
+  describe('GET /map/reverse-geocode', () => {
+    it('should require authentication', async () => {
+      const { status, body } = await request(app).get('/map/reverse-geocode');
+      expect(status).toBe(401);
+      expect(body).toEqual(errorDto.unauthorized);
+    });
+
+    it('should throw an error if a lat is not provided', async () => {
+      const { status, body } = await request(app)
+        .get('/map/reverse-geocode?lon=123')
+        .set('Authorization', `Bearer ${admin.accessToken}`);
+      expect(status).toBe(400);
+      expect(body).toEqual(errorDto.badRequest(['lat must be a latitude string or number']));
+    });
+
+    it('should throw an error if a lat is not a number', async () => {
+      const { status, body } = await request(app)
+        .get('/map/reverse-geocode?lat=abc&lon=123.456')
+        .set('Authorization', `Bearer ${admin.accessToken}`);
+      expect(status).toBe(400);
+      expect(body).toEqual(errorDto.badRequest(['lat must be a latitude string or number']));
+    });
+
+    it('should throw an error if a lat is out of range', async () => {
+      const { status, body } = await request(app)
+        .get('/map/reverse-geocode?lat=91&lon=123.456')
+        .set('Authorization', `Bearer ${admin.accessToken}`);
+      expect(status).toBe(400);
+      expect(body).toEqual(errorDto.badRequest(['lat must be a latitude string or number']));
```

Comment:
Why is this repo method always throwing the same error? That's kinda annoying imo and makes the error a bit useless